### PR TITLE
feat: add distict_id to estates and filter bids for distrincts

### PR DESCRIPTION
**Pull request by nachomazzara**
- Add `distrinct_id` to `Estate` column 
- Set Estate `district_ids` once monitor finished
- Do not index bids for districts
- Do not show bid button on a district details
- Hide `owner of` from Estate details if it is a district

## Note:

`npm run migrate up`

### CODE REVIEW

File: src/Asset/Estate/Estate.model.js [hunk 1]
```diff
@@ -63,4 +64,15 @@ export class Estate extends Model {
         WHERE ${BlockchainEventQueries.byArgs('_estateId', estateId)}
           OR (${BlockchainEventQueries.byArgs('_tokenId', estateId)} AND address = ${address})`)
   }
+
+  static async updateEstateDistrictIds() {
```
Comment (nicosantangelo) [inline src/Asset/Estate/Estate.model.js]: Maybe just `updateDistrictIds` (we're already in Estate.model) or `refreshDistrictIds` because update it's already a verb used in SQL

File: webapp/src/modules/bid/selectors.js [hunk 2]
```diff
@@ -46,6 +46,10 @@ export const getBidByAssetIdFactory = (assetId, assetType) =>
   )
 
 export const getWalletBidsByAsset = (state, asset, assetType) => {
+  if (!asset) {
+    return []
+  }
```
Comment (nicosantangelo) [inline webapp/src/modules/bid/selectors.js]: Why is this needed?

File: src/Asset/Estate/Estate.model.js [hunk 3]
```diff
@@ -63,4 +64,15 @@ export class Estate extends Model {
...
+    return this.db.query(
+      SQL`UPDATE ${SQL.raw(this.tableName)} 
+      SET district_id = P.district_id
+        FROM parcels as P 
+        WHERE ${SQL.raw(this.tableName)}.id = P.estate_id AND
```
Comment (nicosantangelo) [inline src/Asset/Estate/Estate.model.js]: You could also use an alias here

File: webapp/src/modules/bid/selectors.js (hunk 2 above)
Comment (nachomazzara) [reply webapp/src/modules/bid/selectors.js]: Because we are using the selector when you create an Estate and the asset is undefined. I should add a comment. What do you think?

File: src/Asset/Estate/Estate.model.js (hunk 3 above)
Comment (nachomazzara) [reply src/Asset/Estate/Estate.model.js]: `UPDATE` does not work with alias :( if you do something like 

`UPDATE estates as e SET e.district_id = 1` it fails because e is not a valid column from table estates
